examples/multi-backend-multi-mig-bucket-https-lb/main.tf

```
@@ -147,6 +147,8 @@ module "gce-lb-https" {
       enable_cdn                      = false
       session_affinity                = null
       affinity_cookie_ttl_sec         = null
+      custom_request_headers          = null
+
       health_check                    = local.health_check
```

Please run `terraform fmt`.

```suggestion
      health_check = local.health_check
```

---

Thanks!